src/App/Containers/Authentication/SignUp/SignUp.js

```diff
@@ -0,0 +1,85 @@
+import React from 'react';
+import 'bootstrap/dist/css/bootstrap.min.css'
+import './signup.css'
```

Comment:
No need to import it in each and every file if you are importing to the root index or app component.